content/en/apps/guides/data/analytics/setup.md

```diff
@@ -0,0 +1,108 @@
+---
+title: "CHT Sync Setup"
+weight: 100
+linkTitle: "Local Setup"
+description: >
+  Setting up CHT Sync with the CHT
+relatedContent: >
+  core/overview/architecture
+  core/overview/cht-sync
+---
+
+{{% pageinfo %}}
+These instructions apply to both CHT 3.x (beyond 3.12) and CHT 4.x.  
+{{% /pageinfo %}}
+
+Medic maintains CHT Sync which is an integrated solution designed to enable data synchronization between CouchDB and PostgreSQL for the purpose of analytics. It can easily be deployed using Docker. It is supported on CHT 3.12 and later, including CHT 4.x. By using CHT Sync a CHT deployment can easily get analytics by using a data visualization tool. All tools are open source and have no licensing fees.
+
+CHT Sync has been designed to work in both local development environments for testing models or workflows, and in production environments. The setup can accommodate the needs of different stages or environments.
+
+### Prerequisites
+
+- [Docker](https://docs.docker.com/install/)
+- [Docker Compose](https://docs.docker.com/compose/install/)
+- [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+- An `.env` file containing the environment variable. Placeholders can be found in the `env.template` file. The `.env` file should be located in the root directory of the project or set by the operating system. The variables should be customized accordingly for the specific deployment needs.
+
+{{% alert title="Note" %}}
+In order for CHT Sync to transform CouchDB data from the CHT in an easier to query format, it needs to be linked to [CHT Pipeline](https://github.com/medic/cht-pipeline), which contains transformation models for DBT. The schema differs from `couch2pg`. See [`./postgres/init-dbt-resources.sh`](https://github.com/medic/cht-sync/blob/main/postgres/init-dbt-resources.sh).
+{{% /alert %}}
+
+#### Environment variables
```

Comment:
This whole section should be it's own page.  It is 100% shared between production and development, so should not be only on the local development page or only on the prod page.  

also, by removing this section it will greatly simplify the install instructions and reduce the "wall of text" fear that first time readers will have.  Further, we can suggest which _just_ the `.env` vars to change applicable for development.  in the install instructions.